lib/generators/rspec/policy_generator.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Rspec
+  module Generators
+    class PolicyGenerator < ::Rails::Generators::NamedBase
```

Comment:
Let's put all the generators into `action_policy` namespace.
That would allow users to override the templates by creating their own templates under `lib/templates/action_policy/<whatever>`, and we would avoid potential naming conflicts with other gems.

Reply:
I only have to move folders right?